assets/style.css

```diff
@@ -0,0 +1,174 @@
+
+/* Background photos for all screens*/
+.background {
+  background: url(./photos/zodiac.jpg) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+.home-page {
+  background: url(./photos/cosmos.jpg) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+.tarot-page {
+  background: url(./photos/tarot.jpg) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+/* color for nav bar and sun sign buttons */
+.color-aries:hover {
+  background-color: #ffeb3b;
+}
+.color-taurus:hover {
+  background-color: #ff9800;
+}
+.color-leo:hover {
+  background-color: #520748;
+}
+.color-capricorn:hover {
+  background-color: #71c018;
+}
+.color-libra:hover {
+  background-color: #1a8bf5;
+}
+.color-scorpio:hover {
+  background-color: #0d9482;
+}
+.color-cancer:hover {
+  background-color: #af347c;
+}
+.color-sagitarius:hover {
+  background-color: #077446;
+}
+.color-aquarius:hover {
+  background-color: #e29d35;
+}
+.color-pisces:hover {
+  background-color: #b46e06;
+}
+.color-virgo:hover {
+  background-color: #130972;
+}
+.color-gemini:hover {
+  background-color: #bd1a15;
+}
+
+/* styling for mobile nav bar*/
+li {
+  list-style-type: none;
+}
+
+/* pushing all text down from nav bar */
+section {
+  margin-top: 50px !important;
```

Comment:
You would be able to get rid of `!important` if you had this stylesheet imported after the framework import

Using `!important` is almost always the wrong answer